external/stress_tool/st_perf.c

```diff
@@ -40,6 +40,22 @@
 /**
  * Inner Function
  */
+void _smoke_print_interim_result(st_smoke *smoke, st_elapsed_time *duration)
+{
+	char *title = smoke->func->tc_name;
```

Comment:
I think it is better to check whether parameters are NULL or not like below to avoid memory dereferencing.
```
if (!smoke || !smoke->func || duration) {
  return;
}
```

Reply:
it's internal function, so smoke and smoke func can't be null.